pkgs/development/python-modules/autojinja/default.nix

```
@@ -0,0 +1,51 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  jinja2,
+  lib,
+  pytestCheckHook,
+  setuptools,
+  tox,
+  twine,
+}:
+
+buildPythonPackage rec {
+  pname = "autojinja";
+  version = "1.14.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ldflo";
+    repo = "autojinja";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-D+p1EWvtHkOckKwoorQkKWOdpz/PYwYI6V1YzIG+MpU=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    jinja2
+    tox
+    twine
+  ];
```

```suggestion
  dependencies = [ jinja2 ];
```
We only need `jinja2` for the dependencies.
https://github.com/ldflo/autojinja/blob/6ee74ad30f406f9afca2be0d0d4bd209786bd529/setup.py#L38